### PR TITLE
feat(generator): print rendering mode in console

### PR DESCRIPTION
**Pull request by knidarkness**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

Resolve issue #6554 and add information about the build type which is running now.

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [x] I have added tests to cover my changes (if not applicable, please state why)
- [x] All new and existing tests are passing.

### CODE REVIEW

**Review by pimlie — CHANGES_REQUESTED** [review]
Thank you for your contribution!

File: packages/generator/src/generator.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default class Generator {
 
     const routes = await this.initRoutes()
 
-    consola.info('Generating pages')
+    consola.info(`Generating pages for your ${ssr ? 'server-rendered site' : 'SPA'}.`)
```
[the remark this answers is not in the document]
Comment (pi0) [reply packages/generator/src/generator.js]: 1. I agree. We don't need to pass an argument
2. It should be better to rely on `options.render.ssr`
3. I think we are always referring to this mode as `SPA` in docs. Introducing more terms makes it confusing for users IMO.

**Comment by galvez** [comment]
@knidarkness do you plan on continuing work/review on this PR?

**Comment by Atinux** [comment]
Rendering mode will be displayed when target PR is merged: https://github.com/nuxt/nuxt.js/pull/6159

**Pull request closed without merging**